the-collab-lab/tcl-73-smart-shopping-list
# Add Style to Navbar and Page 404

## Description
Enhance the visual appeal of the 404 page and navbar by applying styling. Please note that the color scheme of the navbar text, icons, and background may vary depending on the selected theme and will be implemented additionally.

## Related Issue

N/A

## Acceptance Criteria

N/A


## Type of Changes

enhancement
## Updates


Navbar:

### Before

![navbar1](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/a9839523-62a3-4c35-9021-3bdb20e34477)

### After
![navbae3](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/cda35a5f-fdb6-430b-b8b1-a5a19ab3f8de)

404 page:

### Before
![notfound2](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/f55237b6-aaef-435a-b0fa-2610331e31ba)

### After
<img width="1720" alt="Screenshot 2024-04-01 at 10 34 07 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/8556e147-8741-46bc-bb94-eec289b9a44f">

<img width="1721" alt="Screenshot 2024-04-01 at 10 34 30 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/903ca2b7-c00f-4fd1-98c4-0548b82d1cb3">

## Testing Steps / QA Criteria
git pull
git checkout it-style-navbar-and-page-404 or git checkout it-style-navbar (if you can't access the first one)
npm ci
npm start

- 404 Page:
Navigate to the non-existent route in the app to view styling on the 404 page.
Test the link to confirm if it navigates to the home page and see how changes opacity of the link.

- Navbar:

Navigate to the Home page, List, and Manage. Click on each link to observe the underline and applied style.
Color-related changes will be coordinated with Jose, who is working on the theme in the app.
